doc/API.md

```diff
@@ -1000,341 +1013,333 @@ Response: standard.
-imported to the wallet.
 
-'keyfiles' is a list of filepaths that point to the keyfiles that make up the
-siag key. There should be at least one keyfile per required signature. The
-filenames need to be commna separated (no spaces), which means filepaths that
-contain a comma are not allowed.
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
 
 #### /wallet/lock [POST]
 
-Function: Locks the wallet, wiping all secret keys. After being locked, the
-keys are encrypted. Queries for the seed, to send siafunds, and related queries
-become unavailable. Queries concerning transaction history and balance are
-still available.
+locks the wallet, wiping all secret keys. After being locked, the keys are
+encrypted. Queries for the seed, to send siafunds, and related queries become
+unavailable. Queries concerning transaction history and balance are still
+available.
 
-Parameters: none
-
-Response: standard.
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
 
-#### /wallet/transaction/{id} [GET]
+#### /wallet/transaction/___:id___ [GET]
 
-Function: Get the transaction associated with a specific transaction id.
+gets the transaction associated with a specific transaction id.
 
-Parameters:
+###### Path Parameters [(with comments)](/doc/api/Wallet.md#path-parameters)
 ```
-id string
+id
 ```
-'id' is the ID of the transaction being requested.
 
-Response:
+###### JSON Response [(with comments)](/doc/api/Wallet.md#json-response-7)
+A single transaction contains many objects of the same type. These types are
+listed below.
+```javascript
+// SiacoinInput
+{
+  // TODO: fill this out
+  "parentid":         "1111111111111111111111111111111111111111111111111111111111111111",
+  "unlockconditions": {
+  "timelock":   0,
+  "publickeys": [
+    // TODO: fill this out
+  ],
+  "signaturesrequired": 1
+}
+
+// SiacoinOutput
+{
+  // TODO: fill this out
+}
 ```
-struct {
-	transaction modules.ProcessedTransaction
+
+The response schema is:
+```javascript
+{
+  "transaction": {
+    "transaction": {
+      // TODO: fill this out
+      "siacoininputs": [
+        // See SiacoinInput schema above.
+      ],
+      "siacoinoutputs": [
+        // See SiacoinOutput schema above.
+      ],
+      "filecontracts": [
+        // TODO: fill this out
+      ],
+      "filecontractrevisions": [
+        // TODO: fill this out
+      ],
+      "storageproofs": [
+        // TODO: fill this out
+      ],
+      "siafundinputs": [
+        // TODO: fill this out
+      ],
+      "siafundoutputs": [
+        // TODO: fill this out
+      ],
+      "minerfees": [
+        // TODO: fill this out, why is this an array of types.Currency's?
+      ],
+      "arbitrarydata": [
+        // TODO: is there a restriction on arbitrary data? Why is it an array of array of bytes vs a single byte array?
```

Comment:
is there a restriction on arbitrary data? Why is it an array of array of bytes vs a single byte array?